pcdsdevices/beam_stats.py

```diff
@@ -109,70 +103,80 @@ class Lcls(BaseInterface, Device):
 
     bunch_charge = Cpt(EpicsSignalRO, 'SIOC:SYS0:ML00:AO470', kind='normal',
                        doc='Bunch charge [nC]')
+    bunch_charge_2 = Cpt(EpicsSignalRO, 'Bunch Charge nC: FBCK:BCI0:1:CHRG',
```

Comment:
This looks like a typo:
```suggestion
    bunch_charge_2 = Cpt(EpicsSignalRO, 'FBCK:BCI0:1:CHRG',
```

Reply:
oops thanks Zach,  corrected that in a19476d